### PR TITLE
Removed sign-up via the API

### PR DESCRIPTION
**Pull request by mallok-s**
Removes sign-up command from the CLI

### CODE REVIEW

File: docs/docs/quickstart.ipynb [hunk 1]
```diff
@@ -249,8 +249,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
```
Comment (mands) [inline docs/docs/quickstart.ipynb:252]: I assume this metadata doesn't really matter much

File: projects/python-client/src/datapane/client/api/user.py [hunk 2]
```diff
@@ -153,14 +131,20 @@ def hello_world():
...
+    endpoints as part their respective operations. Keeping the current name is just a convience
+    because we don't need to change any backend code and potentially break signup/login.
     """
 
     def create_token(s: requests.Session) -> str:
-        create_endpoint = furl(path="/api/api-signup-tokens/", origin=server).url
+        create_endpoint = furl(path="/api/api-login-tokens/", origin=server).url
         req = s.post(create_endpoint)
         res: Munch = _process_res(req)
+        print(res)
```
Comment (mands) [inline projects/python-client/src/datapane/client/api/user.py:147]: remove

File: projects/python-client/src/datapane/client/api/user.py [hunk 3]
```diff
@@ -153,14 +131,20 @@ def hello_world():
 
 def token_connect(open_url: str, action: str, server: str) -> t.Optional[str]:
     """
-    Create a signup token, and prompt the user to login/signup while polling for completion.
-    Then log the user into the CLI with the retrieved API token
+    Creates a login token, and prompt the user to login while polling for completion.
+    Then log the user into the CLI with the retrieved API token.
+
+    NOTE: throughout this function we refer to signup tokens however this function will
```
Comment (mands) [inline projects/python-client/src/datapane/client/api/user.py:137]: is this `NOTE` still valid?

**Review by mands — APPROVED** [review]
LGTM - I assume we will need to update the PR in `datapane-hosted` to use this commit / update the submodule upon merging this first?

File: projects/python-client/src/datapane/client/api/user.py (hunk 2 above)
Comment (mallok-s) [reply projects/python-client/src/datapane/client/api/user.py:147]: Done

File: projects/python-client/src/datapane/client/api/user.py (hunk 3 above)
Comment (mallok-s) [reply projects/python-client/src/datapane/client/api/user.py:137]: No, it is no longer valid. I managed to refactor the BE so there are no mentions of sign up tokens

File: docs/docs/quickstart.ipynb (hunk 1 above)
Comment (mallok-s) [reply docs/docs/quickstart.ipynb:252]: No, as far as I am aware it does not matter and is just updated when you make changes the Jupyter Notebook. It doesn't affect how it's rendered. @shahinrostami am I correct in thinking this meta-data doesn't matter?

**Comment by mallok-s** [comment]
> LGTM - I assume we will need to update the PR in `datapane-hosted` to use this commit / update the submodule upon merging this first?

Yes, the `datapane-hosted` PR will needed to be updated to use this commit

**Comment by mands** [comment]
@mallok-s I think we can merge this

**Pull request merged**